test/support/track_sender.ex

```diff
@@ -0,0 +1,84 @@
+defmodule Membrane.RTC.Engine.Support.TrackSender do
+  @moduledoc false
+
+  # This is a very simple version of WebRTC.TrackSender element.
+  #
+  # Engine forwards track starting from a keyframe.
+  # Therefore, whenever some endpoint requests some
+  # track, engine sends keyframe request to the track
+  # source.
+  #
+  # In case of publishing from a file we are not able to
+  # generate keyframe on request. This element ignores
+  # incoming demands and starts sending data after receiving
+  # first keyframe request. Subsequent keyframe requests
+  # are ignored.
```

Comment:
This can be a regular moduledoc - the docs will only be included if `mix docs` is run from `test` MIX_ENV